src/lib/tvgAccessor.cpp

```diff
@@ -70,11 +75,11 @@ unique_ptr<Picture> Accessor::set(unique_ptr<Picture> picture, function<bool(con
 
 Accessor::~Accessor()
 {
-
+    delete(pImpl);
 }
 
 
-Accessor::Accessor()
+Accessor::Accessor() : pImpl(new Impl())
```

Comment:
Good catch but there is no point to alloc memory here...
I suggest you to apply _TVG_DISABLE_CTOR() instead _TVG_DECLARE_PRIVATE()
have a look at Accessor class.